src/app/rundown-view/components/rundown-tooltips/piece-tooltip/piece-tooltip.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, Input } from '@angular/core'
+import { Subject } from 'rxjs'
+import { Tv2PieceType } from 'src/app/core/enums/tv2-piece-type'
+import { PieceLifespan } from 'src/app/core/models/piece-lifespan'
+
+@Component({
+  selector: 'sofie-piece-tooltip',
+  templateUrl: './piece-tooltip.component.html',
+  styleUrls: ['./piece-tooltip.component.scss'],
+})
+export class PieceTooltipComponent {
+  @Input() public playedDurationForPartInMs: number
+  @Input() public type: Tv2PieceType
+  @Input() public isMediaUnavailable: boolean
+  @Input() public pieceLifespan: PieceLifespan
+  @Input() public fileName: string
```

Comment:
`type`, `pieceLifespan` and `filename` should all just be on attribute called `piece: Piece`.

`playedDurationForPartInMs` and `isMediaUnavailable` should be optional because they are only needed for Pieces that need to show hoverscrub in their tooltip, which is not all Pieces that needs to show a Tooltip.

Reply:
Done